### PR TITLE
Feat/component stories

### PR DESCRIPTION
**Pull request by akshaybarya**
Added Stories For Components

### CODE REVIEW

File: stories/ToggleSelection.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  };
+
+  return <ToggleSelection
+    {...args}
+    isActive={isActive}
+    onChange={alterActive}
+  />;
+};
+
+export const Default = Template.bind({});
```
Comment (AdityaVandan) [inline stories/ToggleSelection.stories.tsx:29]: add one more with custom text

File: stories/Tag.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+const Template: Story<TagProps> = (args) => <Tag {...args}>
+  <div className="absolute-center">
+    {tagText}
+  </div>
+</Tag>;
+
+
+export const Warning = Template.bind({});
+
+export const Error = Template.bind({});
```
Comment (AdityaVandan) [inline stories/Tag.stories.tsx:25]: What about Info?

File: stories/Loader.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+import { Story } from '@storybook/react';
+
+import { Loader } from '../src/components/atoms';
+import { DefaultProps as LoaderProps, LOADER_TYPE } from '../src/components/atoms/Loader/Loader';
```
Comment (AdityaVandan) [inline stories/Loader.stories.tsx:6]: use Props instead of default props

File: stories/Tag.stories.tsx (hunk 2 above)
Comment (akshaybarya) [reply stories/Tag.stories.tsx:25]: Currently, info is not deployed once done will update story for it

File: stories/Loader.stories.tsx (hunk 3 above)
Comment (akshaybarya) [reply stories/Loader.stories.tsx:6]: Loader do not have any Prop Type it only has default prop type

File: stories/Tag.stories.tsx (hunk 2 above)
Comment (akshaybarya) [reply stories/Tag.stories.tsx:25]: added story for it in advance

**Review by StTronn — COMMENTED** [review]
Approved

File: stories/ToggleSelection.stories.tsx (hunk 1 above)
Comment (akshaybarya) [reply stories/ToggleSelection.stories.tsx:29]: Added

**Review by AdityaVandan — APPROVED** [review]

**Pull request merged**